.github/workflows/glpi-agentmonitor-ci.yml

```diff
@@ -23,6 +23,10 @@ jobs:
     - name: Compile and link
       run: |
         msbuild GLPI-AgentMonitor.vcxproj -p:Configuration=Release -p:Platform=${{ matrix.arch }} -p:OutDir=Release\ -p:IntermediateOutputPath=Release\ -v:detailed -fl -flp:logfile=Release\msbuild.log
+    - name: Rename built binary to include ${{ matrix.arch }}
+      run: |
+        mv -f "Release\\GLPI-AgentMonitor.exe" "Release\\GLPI-AgentMonitor-${{ matrix.arch }}.exe"
+      shell: bash
```

Comment:
I used bash here as it's available on windows GH image and I did this out of habit. I guess a `ren` can be used in place and merged in previous step.

Reply:
I think this is not too big of a thing to worry about right now. I'll keep it like this and maybe refactor it later...